### ISSUE TITLE
Change port of LocationService

### ISSUE THREAD

**Issue by tboegner** [iris-connect/iris-gateway#181]
LocationService currently uses port 8080 by default which is conflicting with IRIS Client Frontend.
Should be changed to another port that is not in use by any other component.

**Comment by jekutzsche** [comment]
I agree with Timo, but the change needs to be made in many places. @lucky-lusa, what do you think about the ticket?

**Comment by lucky-lusa** [comment]
Since this is only a problem in development, I would suggest overriding the port in your local docker-compose / properties file.

**Comment by lucky-lusa** [comment]
Nevertheless we can think about changing it to a more reasonable port.
